tests/e2e/test_small_file_workload.py

```diff
@@ -0,0 +1,75 @@
+"""
+Test to exercise Small File Workload
+"""
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.utils import get_pod_name
+from ocs_ci.ocs.container import ContainerStor
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, tier1
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def ripsaw(request):
+    # Create Ceph Block Pool
+    sc = ContainerStor(sc_name='ceph-backed', cleanup=True)
+    # Create RipSaw Operator
+    ripsaw = RipSaw()
+    sc.create()
```

Comment:
isn't this creating PVC? the variable name 'sc' is misleading. please correct if my understanding is wrong. 
Also, the comment `Create Ceph Block Pool` is misleading.

Reply:
Yes it is, I will just change the name to pvc,  *create ceph block pool* is to indicate its a rbd backed ?

Reply:
okay. I see that the PVC created here is not used in the test. Do you want to consider using fixture factories instead of ContainerStor class?

Reply:
I can, I will check the fixtures but I want to have my class tested as well in the process for future tests.

Reply:
Fixed, check https://github.com/red-hat-storage/ocs-ci/pull/624